PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!-- DO NOT DELETE THIS TEMPLATE -->
+
+## Checklist
+- [ ] The title of the PR is clear and informative
+- [ ] If applicable, the changes made in the PR have proper test coverage
+- [ ] Issues addressed by the PR are mentioned in the description followed by `Fixes`.
+
+## Description
+
+<!-- Please add a brief description of the changes made in this PR -->
+
+## Fixes
```

Comment:
This is good, maybe we can add one more , e.g. #Issue to refer to the related issue?

Reply:
Fixes section is for listing issues.